ACPs/77-reinventing-subnets/README.md

```diff
@@ -230,36 +237,36 @@ type SubnetValidator struct {
     DisableOwner PChainOwner `json:"disableOwner"`
 }
 
-type ConvertSubnetTx struct {
+type ConvertSubnetToL1Tx struct {
     // Metadata, inputs and outputs
     BaseTx
     // ID of the Subnet to transform
     // Restrictions:
     // - Must not be the Primary Network ID
     Subnet ids.ID `json:"subnetID"`
-    // BlockchainID where the Subnet manager lives
+    // BlockchainID where the validator manager lives
     ChainID ids.ID `json:"chainID"`
-    // Address of the Subnet manager
+    // Address of the validator manager
     Address []byte `json:"address"`
-    // Initial pay-as-you-go validators for the Subnet
-    Validators []SubnetValidator `json:"validators"`
+    // Initial set of validators for the L1
```

Comment:
I think this should be modified to say something similar to continuous-fee-paying. We wouldn't be supplanting the existing validator set with the ones defined in this array.

Not married to "continuous-fee-paying," feel free to modify

```suggestion
    // Initial continuous-fee-paying validators for the L1
```